netbox/users/models.py

```diff
@@ -218,6 +218,7 @@ def clear(self, path, commit=False):
 
 
 @receiver(post_save, sender=User)
```

Comment:
Is this really needed as it seems redundant to me?

Reply:
It is used in a number of tests, as well as Django internals might use it in places (not sure), that would require a deeper dive.

Simply adding the additional recevier doesn't impact much but it will capture any corner cases.

Reply:
What I meant was since `NetBoxUser` is a proxy model to `User`, just having a post save for `NetBoxUser` is not enough?

Reply:
> What I meant was since `NetBoxUser` is a proxy model to `User`, just having a post save for `NetBoxUser` is not enough?

A call to NetBoxUser.save() won't trigger the User.post_save signal (and likewise User.save() won't trigger NetBoxUser.post_save signal) so I don't think so.

We could go through and replace all instances of User with NetBoxUser but if User is used anywhere inside or outside of NetBox code that signal will get missed and it will through an error in the context_processor middleware if the user tries to login without the config built.

I suppose we could also just add a check in the context_processor middleware but I figured this was the better way to handle it.